antlr4/PostgreSQLParser.g4

```diff
@@ -1718,15 +1725,15 @@ non_reserved_keyword
     |  EXCLUDING |  EXCLUSIVE |  EXEC |  EXECUTE |  EXISTS
     |  EXP |  EXPLAIN |  EXTENSION | EXTERNAL |  EXTRACT |  FILTER
     |  FINAL |  FIRST |  FLOAT |  FLOOR |  FOLLOWING
-    |  FORCE |  FORTRAN |  FORWARD |  FOUND |  FREE
+    |  FORCE | FORMAT | FORTRAN |  FORWARD |  FOUND |  FREE
     |  FUNCTION |  FUSION |  G_ |  GENERAL |  GENERATED
     |  GET |  GLOBAL |  GO |  GOTO | GREATEST | GRANTED
     |  GROUPING |  HANDLER |  HIERARCHY |  HOLD | HOST | HOUR
     |  IDENTITY |  IGNORE |  IMMEDIATE |  IMMUTABLE |  IMPLEMENTATION
     |  IMPLICIT |  INCLUDING |  INCREMENT |  INDEX |  INDICATOR
     |  INHERITS |  INOUT |  INPUT |  INSENSITIVE |  INSERT
     |  INSTANCE |  INSTANTIABLE |  INSTEAD |  INT |  INTEGER
-    |  INTERSECTION |  INTERVAL |  INVOKER |  ISOLATION
+    |  INTERSECTION |  INTERVAL |  INVOKER | ISNULL | ISNOTNULL ISOLATION | K_
```

Comment:
If we need `ISNOTNULL` (see comment above), I think you missed a `|` between that and `ISOLATION`. Is there some test that needs it to exist?